pharmd-app/src/screens/student/StudentDrawer.js

```diff
@@ -1,24 +1,23 @@
 /**
  * Description:
- * This component contains the content within the side panel (filters and student preview)
- *     and the side panel itself.
- * TODO:
- *   - Clicking on a student from the student list no longer opens the side bar and no longer opens the student preview expansion panel within the sidebar.
- * Date: 02-06-2021
+ * This Component creates a toggleable sidebar panel with Filters to filter student data and a Quick View containing
+ *     information of a student that the user clicked on in the data grid.
+ *
+ * Date: 03-25-2021
  */
 
 // -------------------------- IMPORTS --------------------------
 
 // Function Imports
 import React, { useState, useEffect } from "react";
-import { useListController } from "react-admin";
-import { useSelector } from "react-redux";
 import { Route, MemoryRouter } from "react-router";
 import { Link as RouterLink } from "react-router-dom";
+import { useListController } from "react-admin";
+import { useSelector } from "react-redux";
 
 // Component Imports
 import DrawerMaterial from "@material-ui/core/Drawer";
-import tw, { styled } from "twin.macro";
+import StudentQuickView from "./StudentQuickView";
```

Comment:
This import is already on line 27. Application fails to compile unless one is removed

Reply:
Good catch!